### PR TITLE
UOE-10364: Remove loginfo feature from Request wrapper

### PR DESCRIPTION
**Pull request by pm-priyanka-bagade**

### CODE REVIEW

File: modules/pubmatic/openwrap/tracker/tracker.go [hunk 1]
```diff
@@ -1,58 +0,0 @@
-package tracker
```
Comment (ShriprasadM) [inline modules/pubmatic/openwrap/tracker/tracker.go:1]: @pm-priyanka-bagade : why aer we deleting entire tracker file here? Is it specific to OW SDK client side bidders?

File: modules/pubmatic/openwrap/auctionresponsehook_test.go [hunk 2]
```diff
@@ -1599,7 +1599,7 @@ func TestOpenWrapHandleAuctionResponseHook(t *testing.T) {
 				mockEngine.EXPECT().RecordNobidErrPrebidServerResponse("5890")
 				mockEngine.EXPECT().RecordPublisherResponseTimeStats("5890", gomock.Any())
 				mockFeature.EXPECT().IsTBFFeatureEnabled(gomock.Any(), gomock.Any()).Return(false).AnyTimes()
-				mockFeature.EXPECT().IsFscApplicable(gomock.Any(), gomock.Any(), gomock.Any()).Return(false)
+				// mockFeature.EXPECT().IsFscApplicable(gomock.Any(), gomock.Any(), gomock.Any()).Return(false)
```
Comment (AvinashKapre) [inline modules/pubmatic/openwrap/auctionresponsehook_test.go:1602]: NitPick: Why did we comment this out? Do we still need it?

File: modules/pubmatic/openwrap/tracker/tracker.go (hunk 1 above)
Comment (pm-priyanka-bagade) [reply modules/pubmatic/openwrap/tracker/tracker.go:1]: Yes , we were using that for logInfo and field was used for client side bidder

File: modules/pubmatic/openwrap/auctionresponsehook_test.go (hunk 2 above)
Comment (pm-priyanka-bagade) [reply modules/pubmatic/openwrap/auctionresponsehook_test.go]: No we don't need this mock

**Review by AvinashKapre — DISMISSED** [review]
LGTM

**Review by pm-saurabh-narkhede — APPROVED** [review]

**Review by AvinashKapre — APPROVED** [review]

**Pull request merged**